### ISSUE TITLE
New tumbleweed logo

### ISSUE THREAD

**Issue by ElFrogster** [openSUSE/distribution-logos#10]
![tumbleweed2](https://github.com/user-attachments/assets/c8b6b0dd-a823-4866-abe4-d3211601f9be)

This is an editable inkscape .svg of my tumbleweed logo submission. Feel free to make any modifications to it or suggest changes.

These are some variants of it that are contained in the file but didn't choose to submit 
![tumbleweed3](https://github.com/user-attachments/assets/cf7ec990-c8b9-4380-a0d5-0a9f0a04da0d)
![tumbleweed4](https://github.com/user-attachments/assets/cf2a95c1-160e-4709-9455-2c2c4a0db4ad)

As for making it a bit less fedora-like. Personally I'd be for keeping the general shape of the infinity symbol. It's made up of fairly basic shapes which is in keeping with the other distribution logo designs. An idea could either be to just flip the logo or do something with the arrows/add other visual elements to make it more distinct

Flipped version
![tumbleweed2](https://github.com/user-attachments/assets/58be59ec-9ba8-4cdd-8788-028baeefb9d2)

**Comment by lkocman** [comment]
Hello folks

I'd say we have to resolve following issues

1) good fit with other distribution logos on Agama and https://get.opensuse.org/
![image](https://github.com/user-attachments/assets/92904583-0a13-4be0-b36f-5c484e24af8e)

2) Ensure that the logo is distinguishable enough from Fedora 

3) Transform all icons in https://github.com/openSUSE/distribution-logos/tree/main/Tumbleweed to the final variant of the logo. See e.g. favicon work on https://github.com/openSUSE/distribution-logos/pull/5

4) @DimStar77 needs to like it :-)

**Comment by hellcp** [comment]
I think this feels a tad complex compared to Leap and MicroOS logos, we should try to aim for a similar level of recognizibility and complexity as the others

**Comment by lkocman** [comment]
Let me start with what I do like about the initial Frogster2 logo (the top most one).

I like the width, compared to the original, color pick is good. I also do like the angle, because it's finally it's square-ish proportions which fit well with others. 

Regarding the box like deco in the middle ... Can we check if any of existing logo elements from other distro logos could be-reused? The current one leaves some sort of leap relation, but then box has a different orientation than the one in Leap logo.

**Comment by ElFrogster** [comment]
> I think this feels a tad complex compared to Leap and MicroOS logos, we should try to aim for a similar level of recognizibility and complexity as the others

Compared to Leap and (maybe) MicroOS, I guess. But I don't see how it's any more complex than Aeon, Slowroll or Leap Micro.

**Comment by lkocman** [comment]
With lcp's fast forward idea in mind ... it will not make the logo less complex :-) but it goes along the fast forward idea a bit ...  Perhaps arrows should have the same size , the smaller one would imply less complexity.

![image](https://github.com/user-attachments/assets/3cf3615f-1343-4414-9a45-8de2ba0b272a)
![image](https://github.com/user-attachments/assets/e393df1a-758e-4311-8986-34ad6a25f872)

**Comment by ElFrogster** [comment]
I actually like the first one quite a bit (although the whitespace between the infinity and outer arrows would need some adjustment).
But the smaller inner arrows might be hard to make out at smaller sizes.

**Comment by ElFrogster** [comment]
![tumbleweed5](https://github.com/user-attachments/assets/ffd63f4e-e9af-4c9c-a417-3d54d8129a9d)

====================================================================================

![tumbleweed6](https://github.com/user-attachments/assets/ceea5497-0b3f-481d-a5ba-7ae4b08d3007)

**Comment by Idesmi** [comment]
I do like the first top one, perhaps with all edges rounded to reflect the style of the other new logos, but it is really too similar to Fedora. I think we either turn the infinite symbol back to horizontal alignment, or add a couple verse arrows on the line.

**Comment by ElFrogster** [comment]
Horizontal alignment would bring the logo back to not being very square-formatted which was seen as an issue with the current one.

Adding arrows onto the lines seems to make it even more fedora-like in my opinion
![tumbleweed8](https://github.com/user-attachments/assets/c0a5c03a-5013-430b-8976-21ecf1dca2ad)

Arrows within the symbol:
![tumbleweed7](https://github.com/user-attachments/assets/d1b3de61-334b-43e1-bc08-85c12fbe5396)

That one looks a bit more like my first submission, which also represented the infinity symbol a bit more differently
![Tumbleweed_frogster](https://github.com/user-attachments/assets/8d6e2f64-f789-41b1-b226-5df29eb62af1)

**Comment by Idesmi** [comment]
I have tried to do a mockup…
Let's overlook my skills in Inkscape. It's the idea that counts ;)
![new_opensuse_tumbleweed_logo_mockup_export-selection](https://github.com/user-attachments/assets/4418c353-872f-4314-89e5-7c968823562d)
To better distinguish ourself, we could add a spiral in the infinite symbol, and keep the arrows somewhere.

**Comment by isbm** [comment]
`<subjective-opinion>`
As I am watching this, I clearly find all these logos together having absolutely no concept, no unity and all of them together looks like a random salad of everything in different colors. It feels like each of these projects are independent standalone something, that simply shares the same OBS/Git repo and nothing else.

But hey. Since there are already so many products and under one umbrella, then why not make a contest of a **set of logos**? Just like one would choose _"icon set for GNOME/KDE"_? This needs to be a concept behind every logo so it is recognisable it comes from openSUSE project.

Each designer participant should think through and integrate all the bits and feelings into EACH logo, so all of them looks like they are all coming from one group. Right now it is just a chaos where openSUSE is losing its iconic identity... 😢 
`</subjective-opinion>`

**Comment by lkocman** [comment]
@isbm you mean like https://en.opensuse.org/Logocontest?

**Comment by lkocman** [comment]
@ElFrogster which variant do you feel the most comfortable with? Both me and @hellcp give you full trust in selection.  Let's just put a lid on it :-)

Just FYI: This will be our refreshed set of backgrounds: (more on the way).
Keep in mind that we'll try to refresh that on more or less anual bassis.
![image](https://github.com/user-attachments/assets/c3dfee01-247f-45f0-b164-99af7e09fc55)

**Comment by isbm** [comment]
> @isbm you mean like https://en.opensuse.org/Logocontest?

No, not quite. The contest seems for a separate logos and it is not clear what for. Why we are changing the logos? Just because we can, or there is a better reason?

Logotype is always (always!) about the _identity_ of a brand. And I think that whatever product you call, e.g. "openSUSE Toaster" or "openSUSE Sausage Grill", I still think that the brand is "openSUSE", which was, is and should be Chameleon.

So I am talking about the design of a whole concept. Say, one takes a topic that OpenSUSE should have a banana as a new identity, so they paint all the logos that hints for a specific product, but feels that it is within the banana topic. Like one style icon set in KDE or GNOME.

The idea is that it should feel like one united concept within a system, rather then a random isolated contributions, where each logo says "I don't care who are my siblings". Currently I don't feel all these products are openSUSE. I feel like all these products are on their own, fully isolated within their own silos with zero communication.

Simply put, if you took a banana as a basis, please integrate that banana in each and every product: slowrolling banana, latest greatest banana, microbanana, desktop banana etc. But people should see a banana at the end, even though that banana could be a different flavour. At the end, the goal must be about **the identity**. If you take any debian stuff, there will be always that iconic swirl. If you take any Ubuntu product, there will be that circle "Ubuntu for humans". And so on.

So at the end, everyone should know "it is openSUSE _something_". Because tomorrow you will invent another few different flavours with a steam engine, guided by a space lasers, but it will be still openSUSE.

Hope it is clear now. 😉

**Comment by ElFrogster** [comment]
> The idea is that it should feel like one united concept within a system, rather then a random isolated contributions, where each logo says "I don't care who are my siblings". Currently I don't feel all these products are openSUSE. I feel like all these products are on their own, fully isolated within their own silos with zero communication.

All the logos do and were asked to follow a clear design principle though. Which is basically "thin lines and basic geometric shapes, using a single colour and usually only strokes with no fill". The Leap Micro logo is a mashup of the Leap and MicroOS logos, the Aeon logo is basically a sliced up and rearranged MicroOS logo. There's definitely some amount of stylistic cohesion.

**Comment by ElFrogster** [comment]
@lkocman 
Honestly, I don't feel super comfortable deciding that entirely on my own, but I can pick out three versions I prefer and then maybe some other people with more stakes in it can give some input on what they prefer.

Version 1 (original proposal, flipped)
![loogo](https://github.com/user-attachments/assets/b4c6ee23-0d1a-476f-97a7-3349c2eaed6f)

Version 2 (abstract arrows within the loops, I think this has probably the most unique shape and looks the least fedora-like)
![logo2](https://github.com/user-attachments/assets/f1c2e3c9-9b4d-48b5-92eb-705c3bdd3dd1)

Version 3 (most direct adaption of the current TW logo, but also fairly fedora like)
![logo7](https://github.com/user-attachments/assets/38e7d52b-6cba-4b1f-9ab8-66eba4fcd010)

Personally, I probably prefer version 2.

**Comment by lkocman** [comment]
Let me just check with Dimstar, otherwise I take it that both me and @hellcp supports your decision. We'll have confirmation tomorrow.

**Comment by lkocman** [comment]
This is how it would look like among others

![image](https://github.com/user-attachments/assets/489c102c-e348-40bf-8618-d8cd871e5a73)

**Comment by TheArcaneBrony** [comment]
Here's some experiments/dirty edits I did with what was posted here before me (just rotated 45deg, emphasizing infinity)

![image](https://github.com/user-attachments/assets/ef56913e-7bb3-4047-82e6-5231ea840569) ![image](https://github.com/user-attachments/assets/b5b0959b-043d-4b2e-a1ba-5e9326a82623) ![image](https://github.com/user-attachments/assets/83cf51a7-5742-40e5-8653-67fee028a291)

**Comment by lkocman** [comment]
@ElFrogster @DimStar77 is okay with your hourglass selection. 

Just one last idea to consider :-) This one would keep the arrows from fast-forward. The middle of 8 adds the second arrow in given direction. Actually it's even more visible in a variant from ElFrogsters comment.


![hourglass2](https://github.com/user-attachments/assets/80098e60-0172-44d4-8c9a-5bfd9a684a61)

**Comment by ElFrogster** [comment]
Yeah, I didn't submit that version as the top three, because I wasn't entirely happy with it. But looking at your version, I do think that's mainly because in my version the arrows are a bit too close together. It looks better with more spacing.

I made some variations (some not entirely cleaned up around the edges), though not entirely sure yet, which one looks best to me (though they are all pretty similar at the end of the day):
![tumbleweed4](https://github.com/user-attachments/assets/08e08934-cc6e-4260-93ad-f10c586a844e)
![tumbleweed5](https://github.com/user-attachments/assets/7db76e6f-ad0c-4974-bf07-0740afe2c0e4)
![tumbleweed6](https://github.com/user-attachments/assets/195dca8b-be4f-4d2a-8723-38d07395ae96)

**Comment by lkocman** [comment]
@hellcp what is your advice? :)

**Comment by hellcp** [comment]
I like them

**Comment by lkocman** [comment]
@ElFrogster let's move to execution then. Let's make a PR :-)

**Comment by ElFrogster** [comment]
Okay, what exactly do I need to do. Just a replacement for suqare-highcolor, or for all of the different ones?

**Comment by ElFrogster** [comment]
Are there any specific requirements in terms of whitespace/padding or similar?

**Comment by hellcp** [comment]
More or less follow what is done for the rest of the logos, there's a very limited description in the README, but it's not really enough

**Comment by lkocman** [comment]
For Leap I literally centered/fitted as close as possible it was to the old logo. Typically overlay in inkscape and work out the detail ...

**Comment by lkocman** [comment]
> Okay, what exactly do I need to do. Just a replacement for suqare-highcolor, or for all of the different ones?
Check discussion in  https://github.com/openSUSE/distribution-logos/pull/5 that might help you.

**Comment by isbm** [comment]
> All the logos do and were asked to follow a clear design principle though. Which is basically "thin lines and basic geometric shapes, using a single colour and usually only strokes with no fill". 

Looks like we are not on the same page, because I am not sure what "thin lines and basic geometric shapes" has at all to do with the identity.

Let me give you an example. The logo below, for example, looks like tamaracas — a rattling percusian instrument, used by an oracle of the Brazilian Tupinamba people:
![image](https://github.com/user-attachments/assets/127a372d-3572-4d8a-9737-bb0353cd811a)

The author probably wanted to highlight the perpetuum of the Tumbleweed using infinity "∞" symbol, so I find this staring at you an African Ostrich looks better:
![image](https://github.com/user-attachments/assets/40334103-60d5-4283-a3f3-edcce53fa2a2)

This one looks like Ostrich on drugs, for example:
![image](https://github.com/user-attachments/assets/b2fe3a6c-ab4f-4c94-bb8f-5e95ce8f8dc3)

For example, long lasting MicroOS logo doesn't resembling anything like "micro" (atoms, fotons, electrons etc comes to mind), but a dude riding a bicycle, wearing a sombrero:
![image](https://github.com/user-attachments/assets/d23469a3-7615-4a74-924d-318ecd561be8)

That one gives me a PTSD flashbacks from Windows 95 times (also SAP users), where we were staring forever at a sandglass mouse cursor, followed by BSOD:
![image](https://github.com/user-attachments/assets/0d7bae03-50d3-4061-9268-d51e16dcc342)

This one reminds me my toaster at the moment it impacted floor, unless it should be read as "a hexagonal wheel, crushing everything":
![image](https://github.com/user-attachments/assets/f715b30f-8029-469f-9e1f-ff8f9eacb98c)

I am sorry. 😉 

For example, if you represent "Aeon" — what exactly you want to say with it? It can be a Greek Deity. It can be a Century. Or Kalpa (buddhist/Hindu cosmology). Or Millenium. Or Saeculum. Or a Japanese company 😉. For example, I would develop that logo, using Greek Mythology. Aion as deity, looks like this:
![image](https://github.com/user-attachments/assets/e9c72b47-7199-4a53-a13a-fa82b728983e)

Here you have it: A Chameleon could have some resemblence of snake around it and placed in "O" circle, as it is usually depicted.

You see, that would be **a concept**. A concept of "Aion Chaemelon" — a Greek God, who controls the lifecycle and cyclic ages (as per Mythology) — that would be a **concept**. This would match the Aeon idea (can by anything else, go ahead with different ideas).

And it would be different from Tumbleweed, which is also "perpetual" and also "until forever" — that on its own a major clash of concepts with Aeon one should address, isn't it? "Perpetuum Chameleon" or "Infinite Chameleon" for Tumbleweed is exactly what should be here. You take a shape of a Chameleon, or a part of it, and then incorporate "∞" symbol. Then it would make perfect sense.

So the bottom line: openSUSE historically has a Chameleon. Chameleon is the center of everything, and then you add "flavours" of it.

As for technique, you can use whatever technique: simple geometrical lines in Inkscape or oil painting in Renaissance style. It doesn't matter. Matters keeping the **identity** of the brand and keeping it as a meaningful concept. However, I personally see that openSUSE is just rapidly losing its iconic identity, which I myself find quite unfortunate and sad. 😢

**Comment by ElFrogster** [comment]
> However, I personally see that openSUSE is just rapidly losing its iconic identity, which I myself find quite unfortunate and sad. 😢

Before the logo change, the tumbleweed logo was an infinity symbol with two arrows. Now it's...an infinity symbol with two arrows, but slightly more stylized.

Before the logo change the Leap logo was a 45° turned square and an arrow. Now it's... a 45° turned square and an arrow.

Main change is that now the logos plus all the new ones are using the same stylistic concepts, so should be easier to identify as belonging to a family.



If you want to make a point the logo needs to have a gecko or similar, sure. But to claim that after the refresh the logos have less of a identity somehow makes zero sense. They follow the same concepts as the old ones, but now with added stylistic cohesion.

**Comment by cboltz** [comment]
I slightly ;-) doubt that switching from a logo that looks like the infinity symbol ("a system you can use forever") to an hourclock ("waiting for the download of the next Tumbleweed snapshot"?) is an improvement. And this doesn't even consider the fact that the hourclock (even if flipped) is quite similar to the Fedora logo. I hope we don't want to tell people "Use Tumbleweed while waiting for the Fedora download to finish" ;-)

The quick drafts TheArcaneBrony posted 3 days ago look like a much better idea to me. They match the style of the other logos, and still remind to the infinity symbol - which is a good match for Tumbleweed.

One detail I don't care too much is the number of arrows - one or two pairs, or even no arrows at all would be fine. But please keep the infinity symbol!

**Comment by lkocman** [comment]
@cboltz please check also #12 which is probably where most comments should happen now.

**Comment by isbm** [comment]
> Before the logo change, the tumbleweed logo was an infinity symbol with two arrows. Now it's...an infinity symbol with two arrows, but slightly more stylized.
> 
> Before the logo change the Leap logo was a 45° turned square and an arrow. Now it's... a 45° turned square and an arrow.

Why you are talking like a syslog? 😉 But hey, at least it proves [the point](https://www.youtube.com/watch?v=dzS-RkzRRVM).

> If you want to make a point the logo needs to have a gecko or similar, sure. But to claim that after the refresh the logos have less of a identity somehow makes zero sense. They follow the same concepts as the old ones, but now with added stylistic cohesion.

I never said that previous logos were any good. ☝🏼 So if there is a contest of changing them, then why not fixing it at the same time? Besides, here I see [many different logos attempts](https://en.opensuse.org/Logocontest), many with quite good ideas to start working with. 🤷🏼‍♂️ 

This one looks like a marine floating bomb (or MicroOS with a chickenpox):
![image](https://github.com/user-attachments/assets/7bd31792-c3a7-4f83-947a-ce6b7985b20e)

But where's Chameleon... 😏

**Comment by tecosaur** [comment]
Having started to see this logo more and more on my system since this change, I feel a need to note that I struggle to see it as anything besides a stylised peanut...

**Comment by lkocman** [comment]
I feel that we should close this issue now when it's implemented. Thank you!
